src/open_inwoner/openzaak/notifications.py

```diff
@@ -287,84 +305,190 @@ def _handle_status_notification(notification: Notification, case: Zaak, inform_u
 
     if not status:
         log_system_action(
-            f"ignored {r} notification: cannot retrieve status {status_url} for case {case.url}",
+            f"ignored {resource} notification: cannot retrieve status {status_url} for case {case.url}",
             log_level=logging.ERROR,
         )
-        return
+        return None
+
+    return status
+
 
+def check_status_type(
+    notification: Notification,
+    case: Zaak,
+    status: Status,
+    oz_config: OpenZaakConfig,
+) -> ZaakType | None:
+    """
+    Check if a status_type exists for `case` and if notifications are enabled
+    """
     status_type = fetch_single_status_type(status.statustype)
+    resource = notification.resource
+
     if not status_type:
         log_system_action(
-            f"ignored {r} notification: cannot retrieve status_type {status.statustype} for case {case.url}",
+            f"ignored {resource} notification: cannot retrieve status_type "
+            f"{status.statustype} for case {case.url}",
             log_level=logging.ERROR,
         )
-        return
+        return None
 
-    if not oz_config.skip_notification_statustype_informeren:
-        if not status_type.informeren:
-            log_system_action(
-                f"ignored {r} notification: status_type.informeren is false for status {status.url} and case {case.url}",
-                log_level=logging.INFO,
-            )
-            return
+    if (
+        not oz_config.skip_notification_statustype_informeren
```

Comment:
I think so, because `oz_config.skip_notification_statustype_informeren` is not checked on its own in either function. In `check_status_type`, it is part of a check that involves `status_type.informeren`, and in `check_zaaktype_config` it's part of a check that also involves `ztc`. So I don't think it is redundant in either case.